### PR TITLE
Prepare Event Hubs October Release

### PR DESCRIPTION
**Pull request by conniey**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by anuchandy — APPROVED** [review]

File: sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 # Release History
 
-## 1.16.0-beta.1 (Unreleased)
+## 1.16.0 (2022-10-13)
```
Comment (ki1729) [inline sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md:3]: Shouldn't we bump the version to 1.15.2 ?

**Review by jsquire — APPROVED** [review]
I cannot verify that all needed updates were made, but the changes look consistent and well-formed.

**Review by ki1729 — APPROVED** [review]

File: sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md (hunk 1 above)
Comment (conniey) [reply sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md:3]: The underlying dependency has a minor bump... revapi was unhappy about it

Comment (vcolin7) [reply sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/CHANGELOG.md:3]: What dependency was this?

**Pull request merged**